docs-devsite/functions.httpscallableoptions.md

```diff
@@ -23,6 +23,7 @@ export interface HttpsCallableOptions
 |  Property | Type | Description |
 |  --- | --- | --- |
 |  [timeout](./functions.httpscallableoptions.md#httpscallableoptionstimeout) | number | Time in milliseconds after which to cancel if there is no response. Default is 70000. |
+|  [useLimitedUseAppCheckToken](./functions.httpscallableoptions.md#httpscallableoptionsuselimiteduseappchecktoken) | boolean | If set to true, uses limited-use App Check token for callable function requests from this instance of [Functions](./functions.functions.md#functions_interface)<!-- -->. You must use limited-use tokens to call functions with replay protection enabled. By default, this is false. |
```

Comment:
And in documentation as well: `limitedUseAppCheckTokens` (note the plural) instead of the double "use".